po/fr/gmajctl.po

```diff
@@ -97,90 +102,103 @@ msgid ""
 "current waveform at two different points giving you a more extreme wide "
 "spread phasing effect."
 msgstr ""
+"Un changement de phase du LFO qui provoque un petit retard dans le point de départ"
+"d'une des formes d'onde. Une fois appliqués aux sorties gauche et droite,"
```

Comment:
et même "in one of the waveform starting points" -> dans l'un des points de départ de la forme d'onde" plutôt.

Reply:
il semble manquer quelque chose dans la version anglaise. 
"When applied to the Left and Right outputs will start the current waveform" @jpcima :
"When applied to the Left and Right outputs, it will start the current waveform" ou
"When applied to the Left and Right, outputs will start the current waveform" peut être ?

Reply:
Ici, y a une erreur dans le texte source (mea culpa)
Il s'agit de "When applied the Left and Right...."
Pas de `to`, ce qui rend la phrase autrement plus sensée.

Reply:
corrigé.